README.md

```diff
@@ -1 +1,22 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
+Project Name
+Hello Microverse
+
+Built With
+HTML and CSS
+No Frameworks
+
+
+Getting Started
+This is an example of how i give instructions on setting up your project locally. i have modified this project, removed sections that don't apply. 
+
+To get a local copy up and running follow these simple example steps.
+
+
+👤 Author1
+
+GitHub: @emmykage
+Twitter: @emmykage
+LinkedIn: Menanya
+👤 Author2
+
```

Comment:
- [ ] Kindly use Microverse template with the formatting to make it looks better you will find it in this [link](https://raw.githubusercontent.com/microverseinc/readme-template/master/README.md) and add the missing sections.